Release notes XLT 7.1.0 (#197)
Xceptance/xlt-documentation: content/en/xlt/release-notes/7_1_x.md

---
title: 7.1.x
linkTitle: 7.1.x

weight: 71
type: docs

date: 2023-07-17

description: >
    Report Enhancements and Fixes

sitemap:
  changefreq: never
  priority: 0.1
---

## XLT 7.1.0

See [here](https://github.com/Xceptance/XLT/milestone/27?closed=1) for a complete list of all changes.

### Test Reports

#### Agent CPU usage chart

> The *Overview* page of a load test report now also shows the *CPU Usage* summary chart for all your agent machines. This allows you to quickly see if there are any problems that may or may not have negatively affected the load test result.

Reviewer: ```suggestion
The *Overview* page of a load test report now also shows the *CPU Usage* summary chart for all your agent machines. This allows you to quickly see if there were any issues that may or may not have negatively affected the load test result.
```

#### Don't filter out summary charts

> When filtering entries in the data table of, say, the *Requests* report page, the corresponding chart groups are filtered as well. Previously, this also affected the summary chart group above the table, but this chart group is no longer filtered out. Same for all other report pages with summary charts.

Reviewer: ```suggestion
When filtering entries in the data table on, say, the *Requests* report page, the filter is also applied to the corresponding groups of charts. Previously, this also affected the summary chart group above the table although it should not. This is fixed now. Summary charts on all report pages will always stay visible regardless of any filter used-
```

> When filtering entries in the data table on, say, the *Requests* report page, the filter is also applied to the corresponding groups of charts. Previously, this also affected the summary chart group above the table although it should not. This is fixed now. Summary charts on all report pages will always stay visible regardless of any filter used-

Reviewer: ```suggestion
When filtering entries in the data table on, say, the *Requests* report page, the filter is also applied to the corresponding groups of charts. Previously, this also affected the summary chart group above the table although it should not. This is fixed now. Summary charts on all report pages will always stay visible regardless of any filter used.
```

> #### Clear button in filter fields overlaps filter text
> 
> Fixed a visual issue in the load test report where the clear button in filter fields overlapped the entered text.

Reviewer: ```suggestion
#### Clear button in filter input fields overlaps filter text

Fixed a visual issue in the load test report where the clear button in filter input fields overlapped the entered text.
```
